File: Print.cpp
// Program to print even or odd
#include <iostream>

int main (void)
{
    int firstPage, lastPage;

    std::cout << "Type fist page: ";
    std::cin >> firstPage;
    std::cout << "Type last page: ";
    std::cin >> lastPage;
    std::cout << "\n";

    int i = firstPage;
    while (i < lastPage && i > 0)
    {
        std::cout << i << ", ";
        i += 2;
    }
    if (i == lastPage)
    {
        std::cout << i << "\n\n";
    }
    return 0;
}

File: Print.java
import java.util.Scanner;

public class Print {
	public static void main(String[] args) {
		Scanner choose = new Scanner(System.in);

		System.out.print("Type first page: ");
		int firstPage = choose.nextInt();

		System.out.print("Type last page: ");
		int lastPage = choose.nextInt();

		int i = 0;
		for (i = firstPage; i < lastPage; i+= 2)
		{
			System.out.print(i + ", ");
		}
		 if (i == lastPage)
		 {
		 	System.out.println(i);
		 }
		choose.close();
	}
}